clifford/test/test_jit_func.py

```diff
@@ -0,0 +1,75 @@
+import unittest
+import numpy as np
+import time
+from ..jit_func import jit_func
+
+
+class TestJITFunc(unittest.TestCase):
```

Comment:
```suggestion
class TestJITFunc:
```
Gets in the way of pytest